ui/src/app/views/workspace/component.html

```diff
@@ -0,0 +1,33 @@
+<h2>AoU Research Portal Workspace</h2>
```

Comment:
Could you litter this and the controller with "TODO"s wherever we're doing something fake? I want to make sure that we don't accidentally miss something that looks slightly real-ish as we fill in actual data (having done that before). For example, "TODO: CDR version info" or "TODO link to {{cohort.notebook}}".